### PR TITLE
Inventory to cloud chamber

### PR DESCRIPTION
**Pull request by manpreet85**
1. Updated Inventory.Yaml to represent 8 racks and each rack having 8 blades.
2. Updating DBInventory.go so that the Inventory is read from the external defintion file.

### CODE REVIEW

File: internal/services/frontend/DBInventory.go [hunk 1]
```diff
@@ -167,11 +167,11 @@ func InitDBInventory() error {
 
 // GetMemoData returns the maximum number of blades held in any rack
 // in the inventory.
-func (m *DBInventory) GetMemoData() (int, int64, *common.BladeCapacity) {
+func (m *DBInventory) GetMemoData() (int64, *common.BladeCapacity) {
```
Comment (Jim3Things) [inline internal/services/frontend/DBInventory.go:170]: The change to add this was made in the recent checkins, and inventory.go depends on it.  Was removing it intentional?

Comment (manpreet85) [reply internal/services/frontend/DBInventory.go:170]: Unintentional. 

Should I reenter and send for PR

Comment (Jim3Things) [reply internal/services/frontend/DBInventory.go:170]: Sure.

Comment (MikeCJohnsonUk) [reply internal/services/frontend/DBInventory.go:170]: Looks like an older version? We're missing the "int" return value in the declaration, and the associated returned value on line 174

File: internal/services/frontend/DBInventory.go [hunk 2]
```diff
@@ -277,3 +277,7 @@ func maxInt64(a int64, b int64) int64 {
 
 	return a
 }
+
+
+//Cloud chamber config file needs to have a setting that point to inventory.yaml file
+//We need dbinventory to create the inventory via the inventory reader
```
Comment (MikeCJohnsonUk) [inline internal/services/frontend/DBInventory.go:283]: Do we need to retain these comments?

**Review by MikeCJohnsonUk — CHANGES_REQUESTED** [review]

**Review by Jim3Things — APPROVED** [review]

**Review by MikeCJohnsonUk — APPROVED** [review]

**Pull request merged**